Sources/HttpResource.swift

```diff
@@ -28,7 +28,10 @@ public struct HttpResource{
 	
 	var uri:String{
 		get{
-			return schema + "://" + host + ":" + port + path
+		        var value = scheme + "://" + host
+		        if let port = port { value += ":" + port }
+		        var value += path
```

Comment:
You'll have to take of the 'var' part of this line, it is throwing a syntax error in the build since you are trying to redefine it. Once you change this we should be able to accept the pull request.